frontend/src/logger/pcm.logger.ts

```diff
@@ -0,0 +1,75 @@
+import {AxiosInstance} from 'axios'
+import {AppConfig} from '../app-config/types'
+import {handleNotAuthorizedErrors} from '../auth/handleNotAuthorizedErrors'
+import identityFn from 'lodash/identity'
+
+enum LogLevel {
+  info = 'INFO',
+  warning = 'WARNING',
+  debug = 'DEBUG',
+  error = 'ERROR',
+  critical = 'CRITICAL',
+}
+
+interface LogEntry {
```

Comment:
suggestion: I would specify both Request and Response types. Something along the lines of:

```ts
interface PostLogsRequest {}
interface PostLogsResponse {}
```

you can then use the `LogEntry` type to detail the request if you wish, and most importantly, use the PostLogsResponse type in your `.catch`